components/post_view/embedded_bindings/select_binding/select_binding.tsx

```diff
@@ -133,15 +158,23 @@ export class SelectBinding extends React.PureComponent<Props, State> {
     }
 
     render() {
+        if (!this.nOptions) {
+            return null;
+        }
+
         const {binding} = this.props;
+        const label = binding.label || binding.location;
+        if (!label) {
+            return null;
+        }
```

Comment:
I'm hesitant to add such fallback on the client side as the server is not longer the single source of truth. Also, the client's verions might be not synchronized and hence there would be different behavior on mobile and webapp.

Reply:
The problem we have at the moment is that for embedded bindings, we have no way to validate in the server side. Therefore, we have to check on the client.

Reply:
Excellent point.